test/src/unit-backwards_compat.cc

```diff
@@ -197,6 +197,11 @@ TEST_CASE(
   query_r.submit();
   array.close();
 
+  // Note: If you encounter a failure here, in particular with a_read[0] == 100
```

Comment:
Is this something we need to address before merging this PR?

Reply:
Nothing to be addressed here.  
Just a hopefully helpful comment to guide those encountering a similar problem in the future as to what to do about it.

Reply:
Resolving...